freenasUI/storage/migrations/0023_auto_20200309_2236.py

```diff
@@ -19,7 +17,7 @@ class Migration(migrations.Migration):
             model_name='replication',
             name='repl_readonly',
             field=models.CharField(choices=[('SET', 'Set to read-only'), ('REQUIRE', 'Require to be read-only'), ('IGNORE', 'Ignore read-only value')],
-                                   default='SET' if get_sw_name().lower() == 'freenas' else 'REQUIRE',
```

Comment:
`if os.path.exists("/data/license")` is how I do it in alembic

Reply:
We have license for certified, which is not truenas (enterprise). Using SET for migrations seems like a fine tradeoff

Reply:
Also in reality 99% of people upgrade in steps, 11.2 -> 11.3 -> 12.0 which would not be a cause for the issue.